src/multio/action/statistics/cfg/StatisticsOptions.cc

```diff
@@ -19,7 +19,8 @@ StatisticsOptions::StatisticsOptions(const config::ComponentConfiguration& compC
     restartLib_{"fstream_io"},
     logPrefix_{"Plan"},
     windowType_{"forward-offset"},
-    accumulatedFieldsResetFreqency_{"month"} {
+    accumulatedFieldsResetFreqency_{"month"},
+    valueCountThreshold_{-1} {
```

Comment:
Should we think about using an `std::optional<long>` for `valueCountThreshold_` instead of assigning and handling negative values?